app/models/sir_trevor_rails/blocks/browse_block.rb

```diff
@@ -32,7 +32,7 @@ def display_item_counts?
       def as_json
         result = super
 
-        result[:data][:item].each do |_k, v|
```

Comment:
I wonder if we should pull `result[:data][:item]` up to a method on the parent class or mixin that returns a "safe" value.

Reply:
Or, an `items` accessor that returns an empty array?

Reply:
I think this also could tie into #1782 (at least a stop-gap fix for #1782)